distribution/debian/scripts/postinst

```diff
@@ -1,21 +1,53 @@
 #!/bin/sh
 set -e
 
-# Add Vector to adm group to read /var/logs
-usermod --append --groups adm vector || true
-
-if getent group 'systemd-journal'
-then
-  # Add Vector to systemd-journal to read journald logs
-  usermod --append --groups systemd-journal vector || true
-  systemctl daemon-reload || true
-fi
-
-if getent group 'systemd-journal-remote'
-then
-  # Add Vector to systemd-journal-remote to read remote journald logs
-  usermod --append --groups systemd-journal-remote vector || true
-  systemctl daemon-reload || true
-fi
+create_user() {
+  id vector >/dev/null 2>&1 ||
+    useradd --system \
+      --shell /sbin/nologin \
+      --home-dir /var/lib/vector \
+      --user-group \
+      --comment "Vector observability data router" \
+      vector
+
+  # Add to adm group to read /var/logs
+  usermod --append --groups adm vector || true
+
+  # Add to systemd-journal to read journald logs
+  if getent group 'systemd-journal'; then
+    usermod --append --groups systemd-journal vector || true
+  fi
+
+  # Add to systemd-journal-remote to read remote journald logs
+  if getent group 'systemd-journal-remote'; then
+    usermod --append --groups systemd-journal-remote vector || true
+  fi
+}
+
+configure_permissions() {
+  if [ ! -d "/var/lib/vector" ]; then
+    # Create default Vector data directory
+    mkdir -p /var/lib/vector
+  fi
+
+  # Own and RW restrict Vector data directory
+  chown -R vector:vector /var/lib/vector
+  chmod 740 /var/lib/vector
```

Comment:
`install -g vector -o vector -m 740 /var/lib/vector` should work ?

Reply:
That's spooky, never seen/used that particular command before - I'm ambivalent, happy to sub it in if that's what's preferred.